src/auryn/Trace.h

```diff
@@ -41,10 +41,12 @@ class Trace : public AurynStateVector
 protected:
 	/*! Decay time constant in [s]. */
 	AurynFloat tau;
+	/* Increment value in inc() */
+	AurynFloat kinc; // Added by ALa
```

Comment:
Update: I tested it now. Speed-wise this change would not be a problem and we can discuss it. It may break the interface though for LinearTrace. Normally all traces can be swapped out with to event-based ones. Still have to think about it.